components/Breadcrumbs.tsx

```diff
@@ -53,7 +53,12 @@ const Breadcrumbs = ({
       }
     })
 
-    setBreadcrumbs(pathArray)
+    setBreadcrumbs(
+      pathArray.slice(
+        0,
+        pathArray.findIndex(breadcrumb => breadcrumb.title === 'lessons') + 1
+      )
+    )
```

Comment:
since you are making a change to this file, why do we need a state for breadcrumbs? Why not just:

```js
const Breadcrumbs = ({...}) => {
  const [_, ...urlPath] = router.asPath.split('/')
  const pathArray = urlPath.map((path, i) => {
      return {
        title: path,
        href: `/${urlPath.slice(0, i + 1).join('/')}`
      }
    })
   const validPaths = ...
```